stack.yaml.lock

```diff
@@ -382,6 +382,20 @@ packages:
       sha256: 05ebc44a9cc5ecda047aa0179020615941ae5b698586854d121b4219ec3cc114
   original:
     url: https://github.com/adjoint-io/poly/archive/b75b853025ea75ba64f62f39d4524832f0117e36.tar.gz
+- completed:
```

Comment:
This .lock file should not be in the commit I think?

Reply:
I think it's helpful, it makes our builds more reproducible